{
    "functionName": "test_convert_fallback_on_wrong_date",
    "className": "TestErrorCases",
    "fileName": "/alexprengere_&_currencyconverter/test_main.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "assert equals(c.convert(10, 'EUR', 'USD', date=date(1986, 2, 2)), 11.789)\npytest.mark.parametrize('c', converters_with_wrong_date_fallback)"
}